boards/arm/nrf5340dk_nrf5340/Kconfig.defconfig

```diff
@@ -20,7 +20,8 @@ config BOARD
 # force building with TF-M as the Secure Execution Environment.
 
 config BUILD_WITH_TFM
-	default y if BOARD_NRF5340DK_NRF5340_CPUAPPNS || BOARD_NRF5340PDK_NRF5340_CPUAPPNS
+        default n
```

Comment:
Could you add a comment for this temp-hack?
Something around this lines 
`Temporarily disable building Non-Secure images with TF-M support by default.`
also, might be an indentation issue here.

Reply:
@oyvindronningstad @anangl , pls, check this out

Reply:
> One alternative option is to remove the whole Kconfig definition of BUILD_WITH_TFM from this file.

I prefer the current form, as it makes the presence of this temp-hack more apparent.